### PR TITLE
Callback doesn't have to be static

### PR DESCRIPTION
**Pull request by patrick-mcdougle**
Adding documentation for symfony/symfony#19745

### CODE REVIEW

File: reference/constraints/Choice.rst [hunk 1]
```diff
@@ -143,6 +143,9 @@ form element.
         }
     }
 
+.. versionadded:: 3.2
+    As of 3.2 the callback no longer needs to be static.
```
Comment (weaverryan) [inline reference/constraints/Choice.rst]: > As of Symfony 3.2...

This is just always the standard we follow

**Comment by weaverryan** [comment]
One minor wording comment, but definitely 👍! Thanks for doc'ing this - we hate to miss new features/changes in the docs :)

Status: Reviewed

**Comment by patrick-mcdougle** [comment]
@weaverryan Updated.

**Comment by xabbuh** [comment]
Thank you @patrick-mcdougle.

**Pull request closed without merging**